web-client/src/presenter/actions/getDocumentSelectedForPreviewAction.ts

```diff
@@ -3,20 +3,7 @@ import { state } from '@web-client/presenter/app.cerebral';
 export const getDocumentSelectedForPreviewAction = ({
   applicationContext,
   get,
-  props,
-}: ActionProps<{
-  documentId?: string;
-}>) => {
-  const { documentId } = props;
-
-  if (documentId) {
```

Comment:
Why was this removed?

Reply:
We no longer need to use the documentId to fetch the document, so this refactor reverts to the original implementation/design of using the tab's file name to fetch the document (docket entry).